geometry_msgs/Vector3 currentCenterOfMass
geometry_msgs/Vector3 targetCenterOfMass
std_msgs/Float64MultiArray homography
